app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -62,14 +62,22 @@ def refresh
         }
         scope :with_content, -> { with_features(PULP_FEATURE, PULP_NODE_FEATURE) }
 
-        def self.default_capsule
+        def self.pulp_master
           unscoped.with_features(PULP_FEATURE).first
```

Comment:
Thats a class method, not an instance method.  I could cache it globally within a threadlocal, but keep in mind it will live there for the life of the server process.

Reply:
Ah, nevermind :)